woocommerce/woocommerce-android
# Product release 1/1953 product shipping shared viewmodel

Closes #1953 - converts the product shipping flow to use the new shared ViewModel (already supported by product detail, pricing, and inventory).

Test A:

- View product detail for a product that has shipping information
- Tap the shipping section
- Edit any of the fields
- Tap Done
- Verify they're reflected in product detail

Test B:

- View product detail for a product that has shipping information
- Tap the shipping section
- Edit any of the fields
- Tap the back button
- After confirming the discard, verify the changes are _not_ reflected in product detail

**Note:** This is a draft because #1954 must be reviewed and merged prior to this one.

![ezgif-6-cfb29f099e95](https://user-images.githubusercontent.com/3903757/74862884-e5548c80-531a-11ea-9982-d985e7b2a59b.gif)


Update release notes:
